packages/serve/src/types.ts

```diff
@@ -17,8 +18,7 @@ export type devServerOptionsType = {
     onAfterSetupMiddleware?: () => void;
     onBeforeSetupMiddleware?: () => void;
     onListening?: () => void;
-    open?: string | boolean | Record<string, unknown>;
-    openPage?: string | string[];
```

Comment:
I think we need `openPage` as currently, we have this object of types as union of v3 and v4.

Reply:
yep lets keep it